invenio_communities/members/services/service.py

```diff
@@ -765,9 +763,9 @@ def request_membership(self, identity, community_id, data, uow=None):
         )
 
         # Create request
-        title = _('Request to join "{community}"').format(
-            community=community.metadata["title"],
-        )
+        title = _('Request to join "%(community)s"') % {
```

Comment:
Does this need the s?

Reply:
Yes, the `s` in %(community)s is necessary because it's part of the string formatting syntax in Python.